server/utils/email.py

```diff
@@ -0,0 +1,73 @@
+
+import sendgrid
+
+from server import app
+from server.models import db, Room, Seat, SeatAssignment
+
+
+def email_students(exam, form):
+    """Emails students in batches of 900"""
+    sg = sendgrid.SendGridAPIClient(api_key=app.config['SENDGRID_API_KEY'])
```

Comment:
Future - we should standardize email api practices

Reply:
This is legacy code (I just moved it to a different file). Will get to this comment once I start building/refactoring the email feature. 

I will not reply to all following email feature comments